facilitator/src/bin/facilitator.rs

```diff
@@ -540,21 +511,13 @@ fn main() -> Result<(), anyhow::Error> {
                         .long("aggregation-start")
                         .value_name("DATE")
                         .help("Beginning of the timespan covered by the aggregation.")
-                        .long_help(
-                            "Beginning of the timespan covered by the \
-                            aggregation. If omitted, the current time is used.",
-                        )
                         .validator(date_validator),
```

Comment:
You didn't mark this one required. Was that intentional?

Reply:
No, just an oversight, thank you for catching that. I tagged this and `aggregation-end` as required.